### ISSUE TITLE
False positive detection in double quoted string containing no variables

### ISSUE THREAD

**Issue by jschaeff** [rodjek/puppet-lint#618]
Hi,

this valid code raises a WARNING from puppet-lint 2.1.0:

```
class {'telegraf':
  global_tags => {
    'role'  => "%{::role}",
  },
}
```

It's an issue because autofix will change the double quotes to single quotes.

**Comment by jschaeff** [comment]
My mistake ...

the valid code is:

```
class {'telegraf':
  global_tags => {
    'role'  => "${::role}",
  },
}
```
